op-e2e/system/e2esys/setup.go

```diff
@@ -810,7 +816,7 @@ func (cfg SystemConfig) Start(t *testing.T, _opts ...SystemConfigOption) (*Syste
 		proposerCLIConfig = &l2os.CLIConfig{
 			L1EthRpc:          sys.EthInstances[RoleL1].UserRPC().RPC(),
 			RollupRpc:         sys.RollupNodes[RoleSeq].UserRPC().RPC(),
-			L2OOAddress:       config.L1Deployments.L2OutputOracleProxy.Hex(),
+			L2OOAddress:       config.L1Deployments(config.AllocTypeL2OO).L2OutputOracleProxy.Hex(),
```

Comment:
Probably also should just use `cfg.AllocType` in case there's more than one alloc type that doesn't use fault proofs in the future and they wind up with the L2OO at different addresses.